src/scripts/extensions/authenticationHelper.ts

```diff
@@ -139,4 +143,43 @@ export class AuthenticationHelper {
 		// Note that we are returning true by default to ensure the N-1 scenario.
 		return userInfo.cookieInRequest !== undefined ? userInfo.cookieInRequest : true;
 	}
+
+	/**
+	 * fetch the user data bounday from the emailAddress
+	 * @param userInfo 
+	 * @returns user data boudary
+	 */
+	private async getUserDataBoundary(userInfo: UserInfoData): Promise<string | undefined> {
```

Comment:
Should this function be taken out as a util to some EUDB utilities file? Since it seems like it is not using any class members, so either it should be static or better take it out from this authenticationhelper class

Reply:
Added a UserDataBoundaryHelper.ts file and extracted this logic to that file